src/host/host.cpp

```diff
@@ -45,17 +45,21 @@ class RPCHandler : virtual public RPCIf {
         assert(argc >= 2);
 
         char *oe_enclave_path = argv[1];
+        uint32_t simulate_flag = OE_ENCLAVE_FLAG_DEBUG;
```

Comment:
We don't need debug mode for the experiment. Also it's no longer a simulate flag, moreso just a flag.

Reply:
Updated flag name to `oe_flag`. Referring to the intel-sgx community: https://community.intel.com/t5/Intel-Software-Guard-Extensions/What-is-the-overhead-of-simulation-mode-VS-NO-SGX-at-all/m-p/1338169, enclaves should be launched in enclave-debug mode.